packages/application-generic/src/services/queues/standard-queue.service.ts

```diff
@@ -21,4 +22,18 @@ export class StandardQueueService extends QueueBaseService {
 
     this.createQueue();
   }
+
+  public async addMinimalJob(data: IStandardJobDto) {
+    return await super.addMinimalJob(data);
+  }
```

Comment:
the `addMinimalJob` is specific just for the `standard queue` do we need to have it in the base?

Reply:
Good point i will remove it from the base.

Reply:
In the end, i removed it altogether. Now we will use add with addMinimalJob data dto.